src/base/QXmppMixIq.cpp

```diff
@@ -46,10 +46,6 @@ QXmppMixIq::QXmppMixIq()
 {
 }
 
-QXmppMixIq::~QXmppMixIq()
```

Comment:
Does this change ABI?

Reply:
Possibly yes ...
I'll revert this change.